### PR TITLE
Activate skylight from ENV var SKYLIGHT_ENABLED

### PR DESCRIPTION
**Pull request by arr-dev**
PR for issue #102 

Should I submit a PR for the [docs repo](https://github.com/skylightio/skylight-docs) as well?

### CODE REVIEW

File: lib/skylight/railtie.rb [hunk 1]
```diff
@@ -117,7 +117,11 @@ def environments
     end
 
     def activate?
-      environments.include?(Rails.env.to_s)
+      environments.include?(Rails.env.to_s) || env_enabled?
+    end
+
+    def env_enabled?
+      ENV.key?("SKYLIGHT_ENABLED") && ENV["SKYLIGHT_ENABLED"] !~ /^false|no$/i
```
Comment (wagenet) [inline lib/skylight/railtie.rb]: I think that if it is explicitly set to `false` then we might want that to override the `environments` list. I'd fall back to the environments list only if the ENV var isn't set. Also, I was looking at our other code for environment variables and we don't handle "no" anywhere. So it's ok to change the regex to just look for "false".

Comment (arr-dev) [reply lib/skylight/railtie.rb]: Fixed and added tests for explicit disable case

**Comment by arr-dev** [comment]
There are some Travis failures, 
but I wasn't able to run `rake run_travis_builds`, always getting Vagrant errors.

I'll try running these versions manually to see if I can replicate it.

**Comment by arr-dev** [comment]
These errors only happen when running full specs, 
but it passes when running `bundle exec rake spec SPEC=spec/integration/rails_spec.rb`

Should I work on resolving this?

**Comment by wagenet** [comment]
I rebased on master, lets see how this goes.

**Comment by wagenet** [comment]
Looks like ordering related failures. I've reproduced locally.

**Comment by wagenet** [comment]
Should be fixed. State wasn't being correctly reset in tests.

**Comment by wagenet** [comment]
@arr-dev a PR for the docs would be great!

**Comment by wagenet** [comment]
@cla-bot check

**Comment by wagenet** [comment]
@cla-bot check

**Comment by arr-dev** [comment]
Added docs PR skylightio/skylight-docs#4

**Pull request merged**